src/atoms/second_order_cone/QolElemAtom.jl

```diff
@@ -30,10 +30,12 @@ function new_conic_form!(context::Context{T}, q::QolElemAtom) where {T}
     x, y = q.children
     t = Variable(x.size)
     for i in 1:length(x)
-        f = vcat(y[i] + t[i], y[i] - t[i], 2 * x[i])
-        add_constraint!(context, GenericConstraint{MOI.SecondOrderCone}(f))
+        f = vcat(t[i], (1 / T(2)) * y[i], x[i])
```

Comment:
@blegat is there any reason to prefer putting `1/2` on `t` or `u`, or `sqrt(2)` on `x`?

Reply:
It probably helps for bigfloat problems

Reply:
Oh sure, which is why I went for `1 / T(2)`. I asked because @blegat originally had `sqrt(2)` on the `x`.

Reply:
Ahh, I missed that context

Reply:
Your solution is better